hal/targets/hal/TARGET_Freescale/TARGET_KSDK2_MCUS/api/objects.h

```diff
@@ -61,6 +61,10 @@ struct dac_s {
     DACName dac;
 };
 
+struct rng_s {
```

Comment:
that's the C HAL convention to avoid having global data in C HAL. Not all HAL implementation require private data, therefore for some platforms it might be empty structure.